src/model.js

```diff
@@ -0,0 +1,45 @@
+/*
+  model.js
+  This file is required. It must export a class with at least one public function called `getData`
+*/
+const ArcgisSearchProviderError = require('./arcgis-search-provider-error');
+const { buildPortalQuery } = require('./helpers/portal-query-builder');
+const { getGeoJson } = require('./helpers/geojson-formatter');
+const { getPortalItems } = require('./helpers/get-items-from-portal');
+const { validateRequestQuery } = require('./helpers/validate-request-query');
+
+const MAX_PAGE_SIZE = 100; // maximum number of results returned from portal per request
+const FIELDS_DEFINITION = require('./fields-definition');
+const PORTAL_URL = 'http://www.arcgis.com/sharing/rest/search';
+
+class ArcgisSearchModel {
+  constructor(options = {}) {
```

Comment:
It should be `constructor(koop, options = {})`

Reply:
changed to `(koop, options = {})`